central/splunk/compliance.go

```diff
@@ -158,6 +174,27 @@ func NewComplianceHandler(complianceDS datastore.DataStore) http.HandlerFunc {
 							}
 						}
 					}
+					for machineKey, machineValue := range d.GetMachineConfigResults() {
+						for controlID, result := range machineValue.GetControlResults() {
+							controlName := controlID
+							if control, ok := controls[controlID]; ok {
+								controlName = control.GetName()
+							}
+							res := &splunkComplianceResult{
+								Standard:   standardName,
+								Cluster:    d.GetDomain().GetCluster().GetName(),
+								ObjectType: "Machine Config",
```

Comment:
It could be a const, yes. I oriented myself on the style of the other loops, where this is also set in place.
I would argue to keep it similar, so if I convert the `ObjectType` to a const, I would do this for all four loops in `compliance.go`.
What do you think?

Reply:
Yes, consistency is more important, let's keep it the way it is. I asked this automatically, but now I see we use this string actually only once in this PR (aside from tests, where it should be verbatim).